modules/storage/storage-account/README.md

```diff
@@ -71,7 +73,7 @@ The output of the module is the ID of the created or existing Storage Account, w
 This example creates a new Storage Account with a unique name in the East US region using the default Storage Account configuration settings. The module output is the ID of the created Storage Account, which can be used in other Azure resource deployments.
 
 ```bicep
-module storageAccount 'br/public:storage/storage-account:2.0.2' = {
+module storageAccount 'br/public:storage/storage-account:0.1.0' = {
```

Comment:
Can you fix the version number? Should be `3.0.1`.

Reply:
updated